.github/workflows/release-binaries.yml

```diff
@@ -0,0 +1,172 @@
+name: Deploy binaries and create release
+on:
+  push:
+    tags:
+      - "v*"
+env:
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: 1
+
+jobs:
+  github_build:
+    name: Build release binaries
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - target: x86_64-unknown-linux-gnu
+            os: ubuntu-latest
+            name: prometheus_wireguard_exporter-x86_64-unknown-linux-gnu.tar.gz
+
+          - target: x86_64-unknown-linux-musl
+            os: ubuntu-latest
+            name: prometheus_wireguard_exporter-x86_64-unknown-linux-musl.tar.gz
+
+          - target: i686-unknown-linux-musl
+            os: ubuntu-latest
+            name: prometheus_wireguard_exporter-i686-unknown-linux-musl.tar.gz
+
+          - target: aarch64-unknown-linux-musl
+            os: ubuntu-latest
+            name: prometheus_wireguard_exporter-aarch64-unknown-linux-musl.tar.gz
+
+          - target: arm-unknown-linux-musleabihf
+            os: ubuntu-latest
+            name: prometheus_wireguard_exporter-arm-unknown-linux-musleabihf.tar.gz
+
+          - target: x86_64-apple-darwin
+            os: macOS-latest
+            name: prometheus_wireguard_exporter-x86_64-apple-darwin.tar.gz
+
+          # - target: aarch64-apple-darwin
+          #   os: macOS-latest
+          #   name: prometheus_wireguard_exporter-aarch64-apple-darwin.tar.gz
+
+          - target: x86_64-pc-windows-msvc
+            os: windows-latest
+            name: prometheus_wireguard_exporter-x86_64-pc-windows-msvc.zip
+
+          - target: i686-pc-windows-msvc
+            os: windows-latest
+            name: prometheus_wireguard_exporter-i686-pc-windows-msvc.zip
+
+          # - target: aarch64-pc-windows-msvc
+          #   os: windows-latest
+          #   name: prometheus_wireguard_exporter-aarch64-pc-windows-msvc.zip
+
+          - target: x86_64-unknown-freebsd
+            os: ubuntu-latest
+            name: prometheus_wireguard_exporter-x86_64-unknown-freebsd.tar.gz
+
+    runs-on: ${{ matrix.os }}
+    continue-on-error: true
```

Comment:
Do we want this? I would rather have it fail and fix it than push only some of the binaries 🤔

Reply:
🤔 valid point.